docs/en/connector-v2/sink/Hive.md

```diff
@@ -37,6 +37,8 @@ By default, we use 2PC commit to ensure `exactly-once`
 | compress_codec                | string  | no       | none           |
 | hdfs_site_path                | string  | no       | -              |
 | hive_site_path                | string  | no       | -              |
+| hive.hadoop.conf              | string  | no       | -              |
+| hive.hadoop.conf-path         | string  | no       | -              |
```

Comment:
Please add the option description in this document.

Reply:
done